### PR TITLE
Bump versions to v2.22.0

### PR DESCRIPTION
**Pull request by AronLin**
#### Highlights

- Support MaskFormer: [Per-Pixel Classification is Not All You Need for Semantic Segmentation](https://arxiv.org/abs/2107.06278) (#7212)
- Support DyHead: [Dynamic Head: Unifying Object Detection Heads with Attentions](https://arxiv.org/abs/2106.08322) (#6823)
- Support [Open Images Dataset](https://storage.googleapis.com/openimages/web/index.html) (#6331)
- Support TIMM backbone: [PyTorch Image Models](https://github.com/rwightman/pytorch-image-models) (#7020)

#### New Features

- Support [MaskFormer](configs/maskformer) (#7212)
- Support [DyHead](configs/dyhead) (#6823)
- Support [OpenImages Dataset](configs/openimages) (#6331)
- Support [TIMM backbone](configs/timm_example) (#7020)
- Support visualization for Panoptic Segmentation (#7041)

#### Breaking Changes

In order to support the visualization for Panoptic Segmentation, the `num_classes` can not be `None` when using the `get_palette` function to determine whether to use the panoptic palette.

#### Bug Fixes

- Fix bug for the best checkpoints can not be saved when the `key_score` is None (#7101)
- Fix MixUp transform filter boxes failing case (#7080)
- Add missing properties in SABLHead (#7091)
- Fix bug when NaNs exist in confusion matrix (#7147)
- Fix PALETTE AttributeError in downstream task (#7230)

#### Improvements

- Speed up SimOTA matching (#7098)
- Add Chinese translation of `docs_zh-CN/tutorials/init_cfg.md` (#7188)

#### Contributors

A total of 20 developers contributed to this release.
Thanks @ZwwWayne, @hhaAndroid, @RangiLyu, @AronLin, @BIGWangYuDong, @jbwang1997, @zytx121, @chhluo, @shinya7y, @LuooChen, @dvansa, @siatwangmin, @del-zhenwu, @vikashranjan26, @haofanwang, @jamiechoi1995, @HJoonKwon, @yarkable, @zhijian-liu, @RangeKing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,10 +74,13 @@ This project is released under the [Apache 2.0 license](LICENSE).
 
 ## Changelog
 
-**2.21.0** was released in 8/2/2022:
+**2.22.0** was released in 24/2/2022:
 
-- Support CPU training
-- Allow to set parameters about multi-processing to speed up training and testing
+- Support MaskFormer
```
Comment (ZwwWayne) [inline README.md:79]: Support MaskFormer, DyHead, and TIMM backbone, then link their config dirs.

File: docs/en/changelog.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
 ## Changelog
 
+### v2.22.0 (24/2/2022)
+
+### Breaking Changes
```
Comment (ZwwWayne) [inline docs/en/changelog.md:5]: '### -> ####'

File: docs/en/changelog.md [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
+
+In order to support the visualization for Panoptic Segmentation, the `num_classes` can not be `None` when using the `get_palette` function to determine whether to use the panoptic palette.
+
+#### New Features
+
+- Support MaskFormer (#7212)
```
Comment (ZwwWayne) [inline docs/en/changelog.md:11]: Add a section of highlight, and link their paper in the changelog.

File: mmdet/__init__.py [hunk 4]
```diff
@@ -16,7 +16,7 @@ def digit_version(version_str):
     return digit_version
 
 
-mmcv_minimum_version = '1.3.17'
+mmcv_minimum_version = '1.4.4'
```
Comment (ZwwWayne) [inline mmdet/__init__.py:19]: We do not need to change this because it is only necessary for CPU training.

File: docs/zh_cn/get_started.md [hunk 5]
```diff
@@ -9,36 +9,37 @@
...
-|      2.4.0       | mmcv-full>=1.1.1, <1.4.0 |
-|      2.3.0       |     mmcv-full==1.0.5     |
-|     2.3.0rc0     |     mmcv-full>=1.0.2     |
-|      2.2.1       |       mmcv==0.6.2        |
-|      2.2.0       |       mmcv==0.6.2        |
-|      2.1.0       |   mmcv>=0.5.9, <=0.6.1   |
-|      2.0.0       |   mmcv>=0.5.1, <=0.5.8   |
+| MMDetection 版本 |          MMCV 版本          |
+|:--------------:|:-------------------------:|
+|     master     | mmcv-full>=1.3.17, <1.5.0 |
```
Comment (hhaAndroid) [inline docs/zh_cn/get_started.md:14]:
```suggestion
|     master     | mmcv-full>=1.4.4, <1.5.0 |
```

File: README.md [hunk 6]
```diff
@@ -74,10 +74,10 @@ This project is released under the [Apache 2.0 license](LICENSE).
 
 ## Changelog
```
Comment (hhaAndroid) [inline README.md:75]: need to modify `model-index.yml`

Comment (hhaAndroid) [reply README.md:75]: The new model needs to be added to the readme file.

File: README.md [hunk 7]
```diff
@@ -74,10 +74,10 @@ This project is released under the [Apache 2.0 license](LICENSE).
...
 
-**2.21.0** was released in 8/2/2022:
+**2.22.0** was released in 24/2/2022:
 
-- Support CPU training
-- Allow to set parameters about multi-processing to speed up training and testing
+- Support [MaskFormer](configs/maskformer), [DyHead](configs/dyhead), [OpenImages Dataset](configs/openimages) and [TIMM backbone](configs/timm_example)
+- Support visualization for Panoptic Segmentation
```
Comment (ZwwWayne) [inline README.md:80]: - Release a good recipe of using ResNet in object detectors pre-trained by [ResNet Strikes Back](https://arxiv.org/abs/2110.00476), which consistently brings about 3~4 mAP improvements over RetinaNet, Faster/Mask/Cascade Mask R-CNN

**Review by ZwwWayne — APPROVED** [review]
